From akpm@linux-foundation.org  Fri Jun  6 15:09:20 2008
From: akpm@linux-foundation.org
Date: Thu, 15 May 2008 12:35:11 -0700
Subject: WUSB: drivers/uwb/wlp/sysfs.c: move misplaced debug statement
To: greg@kroah.com
Cc: akpm@linux-foundation.org, anderson.lizardo@gmail.com
Message-ID: <200805151935.m4FJZBpa006765@imap1.linux-foundation.org>


From: Andrew Morton <akpm@linux-foundation.org>

drivers/uwb/wlp/sysfs.c:142: warning: control may reach end of non-void function 'wlp_wss_neighborhood_print_remove' being inlined

Cc: "Anderson Lizardo" <anderson.lizardo@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/uwb/wlp/sysfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/uwb/wlp/sysfs.c
+++ b/drivers/uwb/wlp/sysfs.c
@@ -125,8 +125,8 @@ ssize_t wlp_wss_neighborhood_print_remov
 
 out:
 	mutex_unlock(&wlp->nbmutex);
-	return used;
 	d_fnend(6, dev, "wlp %p\n", wlp);
+	return used;
 }
 
 
